SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityRunNowRequest.java

```diff
@@ -15,34 +17,38 @@
   private final Optional<List<String>> commandLineArgs;
   private final Optional<Boolean> skipHealthchecks;
   private final Optional<Resources> resources;
+  private final Map<String, String> envOverrides;
   private final Optional<Long> runAt;
 
   public SingularityRunNowRequest(
       Optional<String> message,
       Optional<Boolean> skipHealthchecks,
       Optional<String> runId,
       Optional<List<String>> commandLineArgs,
-      Optional<Resources> resources
+      Optional<Resources> resources,
+      Map<String, String> envOverrides
   ) {
-    this(message, skipHealthchecks, runId, commandLineArgs, resources, Optional.<Long>absent());
+    this(message, skipHealthchecks, runId, commandLineArgs, resources, envOverrides, Optional.<Long>absent());
   }
 
   @JsonCreator
-  public SingularityRunNowRequest(@JsonProperty("message") Optional<String> message,
-                                  @JsonProperty("skipHealthchecks") Optional<Boolean> skipHealthchecks,
-                                  @JsonProperty("runId") Optional<String> runId,
-                                  @JsonProperty("commandLineArgs") Optional<List<String>> commandLineArgs,
+  public SingularityRunNowRequest(@JsonProperty("setMessage") Optional<String> message,
```

Comment:
the `@JsonProperty` on these needs to stay as it was before. This is one of the pieces jackson uses to determine the field name in the json. e.g. `@JsonProperty("message")` becomes `{"message": {}}`

Reply:
Hmmm...think that was my improper use of Intellij's refactoring tools. Going to need to be more careful of that in the future.